package.json

```diff
@@ -22,9 +22,9 @@
   },
   "devDependencies": {
     "LiveScript": "latest",
-    "typescript-compiler": "^1.4",
+    "accord-babel-legacy": "^1.0.0",
     "acorn": "latest",
-    "babel": "latest",
+    "babel": "^5.8.38",
```

Comment:
So this is not going to allow us to test against current babel, which is a larger issue. In order to get both running I might run this through [polytest](https://github.com/carrot/polytest), which we already have wired up in this library...